proxy-node-translator/src/main/java/uk/gov/ida/notification/translator/configuration/CloudHsmSignerConfiguration.java

```diff
@@ -16,12 +18,17 @@
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class CloudHsmSignerConfiguration extends SignerConfiguration {
+
+    private static final Logger LOG = LoggerFactory.getLogger(CloudHsmSignerConfiguration.class);
+
     @JsonCreator
     public CloudHsmSignerConfiguration(
         @JsonProperty("publicKey") DeserializablePublicKeyConfiguration publicKey,
         @JsonProperty("hsmKeyLabel") String hsmKeyLabel
     ) throws SignerConfigurationException {
         try {
+            LOG.info(String.format("Signing eIDAS Response with Cloud HSM using HSM Key Label %s", hsmKeyLabel));
+            LOG.info(String.format("Signing eIDAS Response with Cloud HSM using Public Key %s",publicKey.getCert()));
```

Comment:
Will this just output the subject name of the cert or will it blat the whole thing into the log? Should check the `toString()` of whatever object `getCert()` returns.

Reply:
I was expecting the whole cert. But it will only happen once on startup. Good point though, I'll check.